### PR TITLE
PXC-4311 Release notes for Percona XtraDB Cluster 8.0.34-26

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes/8.0.34-26.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
Good

File: docs/release-notes/8.0.34-26.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+* The `mysql_native_password` authentication plugin is deprecated and may be removed in future releases. Using `CREATE USER`, `ALTER USER`, and `SET PASSWORD` operations, insert a deprecation warning into the server error log if an account attempts to authenticate using `mysql_native_password` as an authentication method.
+
+* The `keyring_file` and `keyring_encrypted_file` plugins are deprecated. These keyring plugins are replaced with the `component_keyring_file` and `component_keyring_encrypted_file` components.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.34 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-34.html).
+
+## Bug fixes
+
+* [PXC-4219](https://jira.percona.com/browse/PXC-4219): A server could enter into a deadlock if `START SLAVE` ran simultaneously when Percona XtraBackup started to back up a server.
```
Comment (kamil-holubicki) [inline docs/release-notes/8.0.34-26.md]: START REPLICA?

File: docs/release-notes/8.0.34-26.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+* [PXC-4238](https://jira.percona.com/browse/PXC-4238): Running `asynchronous_connection_failover_add_source` or `asynchronous_connection_failover_delete_source` UDFs, added an errant transaction, which could prevent a failover in the future.
+
+* [PXC-4284](https://jira.percona.com/browse/PXC-4284): If a MySQL user was not created before the GRANT option, the Percona XtraDB Cluster node was disconnected and needed a complete state transfer (SST).
+
+* [PXC-4288](https://jira.percona.com/browse/PXC-4288): Galera Arbitrator (garbd) used 100% CPU.
+
+* [PXC-4302](https://jira.percona.com/browse/PXC-4302): The GRANT statement could be replicated in a wrong way if `partial_revokes=1` was enabled.
+
+* [PXC-4310](https://jira.percona.com/browse/PXC-4310): There was a wrong link in a warning message.
+
```
Comment (kamil-holubicki) [inline docs/release-notes/8.0.34-26.md:52]: There will be also PXC-4255

Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.34-26.md:52]: added

File: docs/release-notes/8.0.34-26.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.34-26.md]: fixed

**Review by patrickbirch — APPROVED** [review]

**Review by kamil-holubicki — APPROVED** [review]

**Pull request merged**